templates/tower-project.yaml

```diff
@@ -428,12 +421,34 @@ Resources:
               KMSMasterKeyID: !GetAtt EncryptionKeyStack.Outputs.KeyArn
       LifecycleConfiguration:
         Rules:
-          - Id: UserDefinedDataLifecycleRule
-            Status: !Ref EnableScratchLifeCycle
-            ExpirationInDays: !Ref ScratchLifecycleExpiration
+          - Id: LifecycleStorageClassTransition
+            Status: Enabled
             Transitions:
               - TransitionInDays: !Ref ScratchLifecycleTransition
                 StorageClass: !Ref ScratchLifecycleStorageClass
+          - Id: DataExpirationForWorkDirectory
+            Status: !Ref EnableScratchDataExpiration
+            ExpirationInDays: 30
+            Prefix: work/
+          - Id: DataExpirationIn07Days
+            Status: !Ref EnableScratchDataExpiration
+            ExpirationInDays: 7
+            Prefix: 07_days/
```

Comment:
I've been thinking about this some more, and I think I might want to switch to the following categories. 

```
10days
20days
30days
60days
90days
```

Reply:
I am ok with this but, what is your rationale?

Reply:
I feel like people (myself included) would have a harder time remembering `07_days` with both the leading `0` and the underscore, whereas it's easier to remember `10days`.

Reply:
Agreed - I personally also like that it is in multiples of 10.  Easier to remember.  Lets make the announcement with this subtle change - I doubt it will be a big deal.

Reply:
We can make a follow-up announcement once Brad updates the PR next week.

Reply:
I didn't see this comment until today - I agree with these changes and they are included in the latest commit.